pkg/framework/pkcs12.go

```diff
@@ -0,0 +1,74 @@
+// Copyright 2020 Red Hat, Inc. and/or its affiliates
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package framework
+
+import (
+	"crypto/rand"
+	"crypto/x509"
+	"encoding/pem"
+	"errors"
+	v1 "k8s.io/api/core/v1"
+	"software.sslmate.com/src/go-pkcs12"
+	"strings"
+)
+
+const (
+	certificateBlock    = "CERTIFICATE"
+	certificateEndBlock = "-----END CERTIFICATE-----"
+)
+
+// CreatePKCS12TrustStoreFromSecret creates a PCKS12 with certificates inside a Kubernetes secret.
+// The `keys` parameter must contain public certificates only.
+func CreatePKCS12TrustStoreFromSecret(secret *v1.Secret, password string, keys ...string) ([]byte, error) {
```

Comment:
a question, shouldn't we add a -Djavax.net.trustStore proeprties or import t he truststore on the jvm cacerts on the kogito services that will connect to a infinispan with a ssl endpoint? Or will  the truststore be set on propertiesInfinispanQuarkus/Spring props?

Reply:
Truststore is set just for Infinispan client by properties, IMHO there is no need to specify JVM wide truststore.

Reply:
No, there's no need to have this truststore set globally. The purpose of this is to only handle ISPN specific connection.